charts/openshift-metering/templates/hadoop/hadoop-scripts.yaml

```diff
@@ -3,6 +3,14 @@ kind: ConfigMap
 metadata:
   name: hadoop-scripts
 data:
+  azure_secret_copy.sh: |
+    #!/bin/bash
+    set -x
+    cp -v -L -r -f /hadoop-starting-config/* /hadoop-config/
+    export AZURE_PROPERTY_NAME="fs.azure.account.key."$AZURE_STORAGE_ACCOUNT_NAME".blob.core.windows.net"
+    cd /hadoop-config/
+    faq '.configuration.property+=[{name: env.AZURE_PROPERTY_NAME , value: env.AZURE_SECRET_ACCESS_KEY }]' core-site.xml > core-site.xml.temp && cp core-site.xml.temp core-site.xml
```

Comment:
Instead of `&&` just put it onto the next line, and use `set -ex` at the top of the script. `set -e` causes errors to exit the script.